bot.py:22 #INFO     [2024-01-07 19:12:11,672] - __main__ - Starting bot
dispatcher.py:527 #INFO     [2024-01-07 19:12:12,019] - aiogram.dispatcher - Start polling
dispatcher.py:341 #INFO     [2024-01-07 19:12:12,082] - aiogram.dispatcher - Run polling for bot @Adbob_bot id=5021903950 - 'New_Bot'
dispatcher.py:172 #INFO     [2024-01-07 19:12:14,277] - aiogram.event - Update id=269650517 is not handled. Duration 0 ms by bot id=5021903950
dispatcher.py:315 #ERROR    [2024-01-07 19:12:14,277] - aiogram.event - Cause exception while process update id=269650517 by bot id=5021903950
KeyError: 1472177839
Traceback (most recent call last):
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 27, in __call__
    return await handler(event, data)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/fsm/middleware.py", line 41, in __call__
    return await handler(event, data)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 128, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 123, in _wrapped
    return await self._propagate_event(
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 156, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 128, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 123, in _wrapped
    return await self._propagate_event(
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 148, in _propagate_event
    response = await observer.trigger(event, **kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/handlers/user_handlers.py", line 38, in process_beginning_command
    users_db[message.from_user.id]['page'] = 1
KeyError: 1472177839
dispatcher.py:172 #INFO     [2024-01-07 19:12:25,539] - aiogram.event - Update id=269650518 is not handled. Duration 1 ms by bot id=5021903950
dispatcher.py:315 #ERROR    [2024-01-07 19:12:25,539] - aiogram.event - Cause exception while process update id=269650518 by bot id=5021903950
KeyError: 1472177839
Traceback (most recent call last):
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 27, in __call__
    return await handler(event, data)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/fsm/middleware.py", line 41, in __call__
    return await handler(event, data)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 128, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 123, in _wrapped
    return await self._propagate_event(
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 156, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 128, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 123, in _wrapped
    return await self._propagate_event(
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 148, in _propagate_event
    response = await observer.trigger(event, **kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/handlers/user_handlers.py", line 86, in process_forward_press
    if users_db[callback.from_user.id]['page'] < len(book):
KeyError: 1472177839
dispatcher.py:172 #INFO     [2024-01-07 19:12:49,911] - aiogram.event - Update id=269650519 is not handled. Duration 0 ms by bot id=5021903950
dispatcher.py:315 #ERROR    [2024-01-07 19:12:49,911] - aiogram.event - Cause exception while process update id=269650519 by bot id=5021903950
KeyError: 1472177839
Traceback (most recent call last):
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 27, in __call__
    return await handler(event, data)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/fsm/middleware.py", line 41, in __call__
    return await handler(event, data)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 128, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 123, in _wrapped
    return await self._propagate_event(
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 156, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 128, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 123, in _wrapped
    return await self._propagate_event(
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 148, in _propagate_event
    response = await observer.trigger(event, **kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/virtul/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/Users/aleksandrdubov/Documents/Программирование/Bookbot/handlers/user_handlers.py", line 38, in process_beginning_command
    users_db[message.from_user.id]['page'] = 1
KeyError: 1472177839
dispatcher.py:172 #INFO     [2024-01-07 19:13:03,237] - aiogram.event - Update id=269650520 is handled. Duration 327 ms by bot id=5021903950
dispatcher.py:172 #INFO     [2024-01-07 19:13:04,727] - aiogram.event - Update id=269650521 is handled. Duration 105 ms by bot id=5021903950
dispatcher.py:172 #INFO     [2024-01-07 19:13:06,247] - aiogram.event - Update id=269650522 is handled. Duration 142 ms by bot id=5021903950
dispatcher.py:172 #INFO     [2024-01-07 19:13:08,533] - aiogram.event - Update id=269650523 is handled. Duration 400 ms by bot id=5021903950
dispatcher.py:455 #WARNING  [2024-01-07 19:20:00,454] - aiogram.dispatcher - Received SIGINT signal
dispatcher.py:359 #INFO     [2024-01-07 19:20:00,456] - aiogram.dispatcher - Polling stopped for bot @Adbob_bot id=5021903950 - 'New_Bot'
dispatcher.py:554 #INFO     [2024-01-07 19:20:00,456] - aiogram.dispatcher - Polling stopped
